static/js/search/statvar_results.tsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Result section for matching stat vars
+ */
+
+import _ from "lodash";
+import React, { useState } from "react";
+
+import { NamedNode } from "../shared/types";
+
+const NUM_EXAMPLE_SV = 10;
+const REDIRECT_PREFIX = "/tools/statvar#";
+
+interface StatVarResultsProps {
+  statVars: NamedNode[];
+}
+
+export function StatVarResults(props: StatVarResultsProps): JSX.Element {
+  const [showAll, setShowAll] = useState(false);
+
+  if (_.isEmpty(props.statVars)) {
+    return <></>;
+  }
+  const shownStatVars = showAll
+    ? props.statVars
+    : props.statVars.slice(0, Math.min(props.statVars.length, NUM_EXAMPLE_SV));
+  const canExpand = props.statVars.length > NUM_EXAMPLE_SV && !showAll;
+  return (
+    <div className="search-results-sv search-results-section">
+      <div className="search-results-section-title">Statistical Variables</div>
+      <div className="search-results-section-content">
+        <div className="search-results-list">
+          {shownStatVars.map((sv) => {
+            return (
+              <div key={`sv-result-${sv.dcid}`} className="search-results-item">
+                <a
+                  href={REDIRECT_PREFIX + sv.dcid}
+                  className="search-results-item-title"
+                >
+                  {sv.name}
+                </a>
+                <div className="search-results-item-byline">
+                  dcid: {sv.dcid}
+                </div>
+              </div>
+            );
+          })}
+        </div>
+        {canExpand && (
+          <div
+            onClick={() => setShowAll(true)}
+            className="search-results-sv-show-more"
+          >
+            show more
+          </div>
+        )}
+        {showAll && (
+          <div
```

Comment:
In this mode, should we go full width of the results container?

Reply:
I'm curious what the advantages of going the full width of the results container would be? currently, users don't need to scroll horizontally & when clicking show more on a list, I don't think expanding horizontally makes as much sense as expanding vertically

Reply:
ah, i meant to expand both vertically and horizontally (or rather, we can keep it full width always). it seems awkward that it's half-width. we could update this in another iteration too

Reply:
ok, will do this in next iteration